rules/xcodeproj.bzl

```diff
@@ -806,6 +808,7 @@ Tags for configuration:
         If not present the 'Debug' and 'Release' Xcode build configurations will be created by default without
         appending any additional bazel invocation flags.
         """),
+        "custom_lldbinit": attr.string(mandatory = False, default = "", doc = "Custom path to the .lldbinit file"),
```

Comment:
why does this need to be customizable? rules_ios is already writing the settings https://github.com/bazel-ios/rules_ios/blob/c8287ff5f1bd9458f075b79849103bedaa5526e4/tools/xcodeproj_shims/install.sh#L103, could the scheme & that file coordinate on a path (e.g. `$CONFIGURATION_TEMP_DIR/$TARGET_NAME.lldbinit`) so it "just works" out of the box?

Reply:
That makes sense, went on a "natural" route of the user might want to specify this. But in reality having it "just work" is a better approach IMO.

Will push some changes with this in mind!

Reply:
You can look for references to `bazelinstallers` here. We have some installers (like lldb-settings.sh, swiftmodules.sh, etc.) You can create a new one called `lldbinit` and place it either inside or next to the same bazelinstallers directory. And then the scheme_action_details can reference either the `$PROJECT_FILE_PATH/bazelinstallers/lldbinit` or `$PROJECT_FILE_PATH/lldbinit`.

Reply:
Note that you can generate the lldbinit file from a template, in the same way we generate the `print_json_leaf_nodes` file and several others ... we basically just copy these files from our xcodeproj_shims into the newly created xcode project directory.

Reply:
🙏